.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ on:
       - 'FILE_HEADER'
 
 env:
-  IMAGE:                           paritytech/ci-unified:bullseye-1.74.0
+  IMAGE:                           paritytech/ci-unified:bullseye-1.75.0
```

Comment:
I believe we should keep the image aligned across workflows. It is also used in .github/workflows/measurements.yml.